.github/workflows/asan.yml

```diff
@@ -0,0 +1,70 @@
+name: Address Sanitizer analysis
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - main
+  schedule:
+    - cron: '0 1 * * *'
+
+jobs:
+
+  asan-test:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Sync eProsima/Fast-DDS repository
+        uses: actions/checkout@v3
+        with:
+          path: src/Fast-DDS
+
+      - name: Install apt packages
+        uses: ./src/Fast-DDS/.github/actions/install-apt-packages
+
+      - name: Install GTest
+        uses: ./src/Fast-DDS/.github/actions/install-gtest-linux
+
+      - name: Install Python packages
+        uses: ./src/Fast-DDS/.github/actions/install-python-packages
+
+      - name: Update colcon mixin
+        run: |
+          colcon mixin add default \
+            https://raw.githubusercontent.com/colcon/colcon-mixin-repository/master/index.yaml
+          colcon mixin update default
+        continue-on-error: true
```

Comment:
Is this necessary?

Reply:
Removed as we do not use the colcon mixins